firestore-counter/functions/__tests__/util.ts

```diff
@@ -0,0 +1,137 @@
+/*
```

Comment:
As this is essentially taken from the node client. Was this recreated as we cannot reference the current client in the `clients` folder?

Maybe add a comment to reference the original source? Would this filename be better as `client.ts` or `test_client.ts` as it does not include any other utils?